CONTRIBUTING.md

```diff
@@ -2,8 +2,18 @@
 
 Each proposal to Mainnet (agoric-3) should be contributed to this repo before being proposed to the chain.
 
-1. Prepare the proposal and verify it running locally
+1. Prepare the proposal and verify it by creating a proposal in
```

Comment:
please link to it. it's stable enough: https://github.com/Agoric/agoric-sdk/blob/master/a3p-integration/README.md

Reply:
done